03_letters/mohitnag/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLetters(t *testing.T) {
+	type testCases struct {
+		input    string
+		expected map[rune]int
+	}
+	testData := map[string]testCases{
+		"Happy Case": {"eaa!baabq23123!",
+			map[rune]int{33: 2, 49: 1, 50: 2, 51: 2, 97: 4, 98: 2, 101: 1, 113: 1}},
```

Comment:
Here this isn't complete. You are still assigning the values for the keys in the maps as int representations. Use the actual characters, this improves readability by a mile.

If you know you have an input string of "aaaabbbb" then I would expect an output of `map[rune]int{'a':3, 'b':3}` that is so much easier to read than the int representation of the rune.

Reply:
Agree. Done